policy/manager.go

```diff
@@ -43,16 +42,18 @@ func NewManager(log hclog.Logger, ps Source, pm *manager.PluginManager) *Manager
 func (m *Manager) Run(ctx context.Context, evalCh chan<- *Evaluation) {
 	defer m.stopHandlers()
 
-	policyIDsCh := make(chan []PolicyID)
-	policyIDsErrCh := make(chan error)
+	policyIDsCh := make(chan IDMessage, 2)
+	policyIDsErrCh := make(chan error, 2)
```

Comment:
I thought it useful to allow policies sources to send at least 1 message each before getting blocked on the send.

Reply:
Ah ok. I don't think it would make much difference though, since there's no guarantee the two messages would come from different sources 🤔 

But I think it's fine to leave it like this 👍